x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/process/MemoryUsageEstimationProcessManager.java

```diff
@@ -66,7 +67,9 @@ private MemoryUsageEstimationResult runJob(String jobId,
             new AnalyticsProcessConfig(
                 dataSummary.rows,
                 dataSummary.cols,
-                DataFrameAnalyticsConfig.MIN_MODEL_MEMORY_LIMIT,
+                // For memory estimation the model memory limit here should be set high enough not to trigger an error when C++ code
+                // compares the limit to the result of estimation.
+                new ByteSizeValue(1, ByteSizeUnit.TB),
```

Comment:
Unless there is a side-effect, I wonder if we should make this even larger. The era of `1 TB` RAMs is around the corner :-)

Reply:
Done.